_includes/common/ui/transactions-page.md

```diff
@@ -0,0 +1,102 @@
+{% if page.cloud == true %}
+  {% capture link_prefix %}../{{site.versions["stable"]}}/{% endcapture %}
+  {% assign page_prefix = "" %}
+{% else %}
+  {% assign link_prefix = "" %}
+  {% assign page_prefix = "ui-" %}
+{% endif %}
+
+The **Transactions** page helps you:
+
+- Identify frequently retried or high latency transactions.
+- View transaction [details](#transaction-details-page).
+
+{{site.data.alerts.callout_success}}
+In contrast with the [**Statements** page]({{ page_prefix }}statements-page.html), which displays [SQL statement fingerprints]({{ page_prefix }}statements-page.html#sql-statement-fingerprints), the **Transactions** page displays SQL statement fingerprints grouped by [transaction]({{ link_prefix }}transactions.html).
+{{site.data.alerts.end}}
+
+{% if page.cloud != true %}
+To view this page, [access the DB Console](ui-overview.html#db-console-access) and click **Transactions** in the left-hand navigation.
+{% else %}
+To view this page, click **SQL Activity** in the left-hand navigation of the {{ site.data.products.db }} Console and then click the **Transactions** tab.
+{% endif %}
+
+## Search and filter by application
+
+By default, this page shows transactions from all applications running on the cluster, and hides internal CockroachDB transactions.
+
+To filter the transactions by [`application_name`]({{ link_prefix }}connection-parameters.html#additional-connection-parameters), use the **App** pulldown in the **Filters** menu. If you haven't set `application_name` in the client connection string, it appears as `unset`.
+
+- CockroachDB's internal transactions are only displayed under the `$ internal` app.
+- Transactions from the SQL shell are displayed under the `$ cockroach sql` app.
```

Comment:
You can also fitler by Query fingerprint runs longer than a time period. See the description in the Statement page.